packages/swirl-components/src/components/swirl-card/swirl-card.tsx

```diff
@@ -15,6 +15,7 @@ export class SwirlCard {
   @Prop() interactive?: boolean;
   @Prop() href?: string;
   @Prop() linkTarget?: string;
+  @Prop() highlightActive?: boolean;
```

Comment:
**nitpick:** We typically write these kinds of props as adjectives, `highlighted` in this case.